src/homeWork/module5/task2/Passport.java

```diff
@@ -0,0 +1,54 @@
+package homeWork.module5.task2;
+
+/**
+ * Created by : Volodymyr_Silitskyi
+ * Created at : 6/12/2018
+ */
+
+
+public class Passport {
```

Comment:
Please look at the code convention rules, in which order class members should appear in the class.

1. Class (static) variables: First the public class variables, then the protected, and then the private.
2. Instance variables: First public, then protected, and then private.
3. Constructors
4. Methods: These methods should be grouped by functionality rather than by scope or accessibility. For example, a private class method can be in between two public instance methods. The goal is to make reading and understanding the code easier.

Source: http://www.oracle.com/technetwork/java/codeconventions-141855.html